<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Counter</class>
 <widget class="QMainWindow" name="Counter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bottle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>321</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="oneoo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>200</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>+ 100 мл.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twofiveo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>200</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>+ 250 мл.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveoo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>200</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>+ 500 мл.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="litre">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>470</y>
      <width>200</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>+ 1 л.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Необходимое количество воды:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Запись в Дневник.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Made by:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Здесь могла быть ваша реклама.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
